From: Mirco Bauer <meebey@meebey.net>
Date: Sun, 5 Apr 2009 20:34:13 +0000 (+0200)
Subject: set subdir variable to "po" in all po-*/Makefile.in.in files
X-Git-Url: http://git.qnetp.net/?p=smuxi.git;a=commitdiff_plain;h=81669b0fcb404562d9d0da99d9605b3f64c07af2

set subdir variable to "po" in all po-*/Makefile.in.in files

This doesn't really make sense to me as that causes to always stamp the same file (po/stamp-it) instead of each po directory, but this is how intltoolize wants it.
At the same time this finally fixes the endless build loop if the source files are laying on xfs or tmpfs (which support more precise timestamps). (closes: #156)
---

diff --git a/po-Engine-IRC/Makefile.in.in b/po-Engine-IRC/Makefile.in.in
index 27268fc..3add1c1 100644
--- a/po-Engine-IRC/Makefile.in.in
+++ b/po-Engine-IRC/Makefile.in.in
@@ -35,7 +35,7 @@ datarootdir = @datarootdir@
 libdir = @libdir@
 DATADIRNAME = @DATADIRNAME@
 itlocaledir = $(prefix)/$(DATADIRNAME)/locale
-subdir = po-Engine-IRC
+subdir = po
 install_sh = @install_sh@
 # Automake >= 1.8 provides @mkdir_p@.
 # Until it can be supposed, use the safe fallback:
diff --git a/po-Engine/Makefile.in.in b/po-Engine/Makefile.in.in
index b84aa3a..2ec381e 100644
--- a/po-Engine/Makefile.in.in
+++ b/po-Engine/Makefile.in.in
@@ -35,7 +35,7 @@ datarootdir = @datarootdir@
 libdir = @libdir@
 DATADIRNAME = @DATADIRNAME@
 itlocaledir = $(prefix)/$(DATADIRNAME)/locale
-subdir = po-Engine
+subdir = po
 install_sh = @install_sh@
 # Automake >= 1.8 provides @mkdir_p@.
 # Until it can be supposed, use the safe fallback:
diff --git a/po-Frontend-GNOME-IRC/Makefile.in.in b/po-Frontend-GNOME-IRC/Makefile.in.in
index 70d5713..5f70427 100644
--- a/po-Frontend-GNOME-IRC/Makefile.in.in
+++ b/po-Frontend-GNOME-IRC/Makefile.in.in
@@ -35,7 +35,7 @@ datarootdir = @datarootdir@
 libdir = @libdir@
 DATADIRNAME = @DATADIRNAME@
 itlocaledir = $(prefix)/$(DATADIRNAME)/locale
-subdir = po-Frontend-GNOME-IRC
+subdir = po
 install_sh = @install_sh@
 # Automake >= 1.8 provides @mkdir_p@.
 # Until it can be supposed, use the safe fallback:
diff --git a/po-Frontend-GNOME/Makefile.in.in b/po-Frontend-GNOME/Makefile.in.in
index 1e74fca..9567482 100644
--- a/po-Frontend-GNOME/Makefile.in.in
+++ b/po-Frontend-GNOME/Makefile.in.in
@@ -35,7 +35,7 @@ datarootdir = @datarootdir@
 libdir = @libdir@
 DATADIRNAME = @DATADIRNAME@
 itlocaledir = $(prefix)/$(DATADIRNAME)/locale
-subdir = po-Frontend-GNOME
+subdir = po
 install_sh = @install_sh@
 # Automake >= 1.8 provides @mkdir_p@.
 # Until it can be supposed, use the safe fallback:
diff --git a/po-Server/Makefile.in.in b/po-Server/Makefile.in.in
index 925fe32..edffc6e 100644
--- a/po-Server/Makefile.in.in
+++ b/po-Server/Makefile.in.in
@@ -35,7 +35,7 @@ datarootdir = @datarootdir@
 libdir = @libdir@
 DATADIRNAME = @DATADIRNAME@
 itlocaledir = $(prefix)/$(DATADIRNAME)/locale
-subdir = po-Server
+subdir = po
 install_sh = @install_sh@
 # Automake >= 1.8 provides @mkdir_p@.
 # Until it can be supposed, use the safe fallback:
